docs/_includes/split-button.html

```diff
@@ -0,0 +1,240 @@
+<div id="split-button">
+    {% include section-header.html name="split-button" version=page.versions.split-button %}
+
+    <p>Split button is a button with a menu-button squished together.<p>
+    
+        <div class="demo">
+            <div class="demo__inner">
+                <button class="btn btn--primary split-button" aria-haspopup="true" type="button">
```

Comment:
So this syntax is contrary to `bem` from what I remember.
Either we do:
`split-button` as it's own component without a depdency on btn (so import btn styles in split button component)
or have split button be a modifier on `btn` like `btn--split-button`
I prefer the first one where split button is it's own component. Make btn styles imported in split btn.

Reply:
Couple of approaches I can think of:

1 Create new `split` modifiers for btn and menu-button. Create a new top level `split-button` class. This means a CSS dependency on btn and menu-btn modules (which shouldn't be a problem in my opinion).

```html
<div class="split-button">
    <button class="btn btn--primary btn--split"></button>
    <span class="menu-btn menu-btn--primary menu-btn--split"></span>
    <div class="menu-btn__menu"></div>
</div>
```

or

2 Complete standalone, no CSS dependencies. This would require mixins for `split-button__btn` and `split-button__menu-btn` etc.  and all their associated modifiers. Could get a little mixin heavy in the code.

```html
<div class="split-button">
    <button class="split-button__btn split-button__btn--primary"></button>
    <span class="split-button__menu-btn split-button__menu-btn--primary"></span>
    <div class="split-button__menu"></div>
</div>
```